CMSIS/NN/Source/NNSupportFunctions/arm_q7_to_q15_no_shift.c

```diff
@@ -30,81 +30,71 @@
 
 #include "arm_nnsupportfunctions.h"
 
-/**    
- * @ingroup groupSupport    
+/**
+ * @ingroup groupSupport
  */
 
-/**    
- * @addtogroup nndata_convert    
- * @{    
+/**
+ * @addtogroup nndata_convert
+ * @{
  */
 
-/**    
- * @brief Converts the elements of the Q7 vector to Q15 vector without left-shift 
- * @param[in]       *pSrc points to the Q7 input vector    
- * @param[out]      *pDst points to the Q15 output vector   
- * @param[in]       blockSize length of the input vector    
- * @return none.    
- *    
- * \par Description:    
- *    
- * The equation used for the conversion process is:    
- *   
- * <pre>    
- * 	pDst[n] = (q15_t) pSrc[n];   0 <= n < blockSize.    
- * </pre>    
- *   
+/**
+ * @brief Converts the elements of the Q7 vector to Q15 vector without left-shift
+ * @param[in]       *pSrc points to the Q7 input vector
+ * @param[out]      *pDst points to the Q15 output vector
+ * @param[in]       blockSize length of the input vector
+ * @return none.
+ *
+ * \par Description:
+ *
+ * The equation used for the conversion process is:
+ *
+ * <pre>
+ * 	pDst[n] = (q15_t) pSrc[n];   0 <= n < blockSize.
+ * </pre>
+ *
  */
 
 void arm_q7_to_q15_no_shift(const q7_t * pSrc, q15_t * pDst, uint32_t blockSize)
 {
-    const q7_t *pIn = pSrc;     /* Src pointer */
-    uint32_t  blkCnt;           /* loop counter */
+    const q7_t *pIn = pSrc;
+    uint32_t  blkCnt;
 
 #ifndef ARM_MATH_CM0_FAMILY
```

Comment:
We are no more using this flag in arm_math.h and CMSIS-DSP.

Instead I would use:

#if defined(ARM_MATH_LOOPUNROLL) && defined (ARM_MATH_DSP)

Because the loop is obviously unrolled and contained intrinsics

Reply:
ok..got it. I'll update the patch and correct this in other files in future patches..